apps/builder/app/builder/features/style-panel/sections/transitions/transition-property.tsx

```diff
@@ -183,19 +215,29 @@ export const TransitionProperty = ({
               <ComboboxScrollArea>
                 {isOpen && (
                   <>
-                    <ComboboxLabel>Common</ComboboxLabel>
-                    {commonProperties.map(renderItem)}
+                    {propertiesOnInstance.length > 0 && (
```

Comment:
@TrySound what do we call propertied defined on an instance, including local, token and states?

Reply:
cascaded values

Reply:
defined is fine too

Reply:
defined sounds good